devconf-outreachy/source/index.rst

```diff
@@ -0,0 +1,344 @@
+
+Becoming OpenStack Developer via Outreachy
+==========================================
+
+Presented by: Aija Jauntēva, Dmitry Tantsur, Richard Pioso, Ilya Etingof
+
+In this talk
```

Comment:
@dtantsur @ajya Guys, I think I am almost done recombining/adding slides and notes. Also, I've tried to split the talk onto more or less equal parts.

Your feedback and n/ACKs please ;-)

Reply:
ACK, thanks for all the work, 
created a PR